### PR TITLE
Post-testing GSSAPI edits

### PR DESCRIPTION
**Pull request by Amruta-Ranade**

### CODE REVIEW

**Comment by Amruta-Ranade** [comment]
___
*[v19.1/gssapi_authentication.md, line 84 at r1](https://reviewable.io/reviews/cockroachdb/docs/4562#-LaXILlKAn8_YnFgLOYF:-LaXRFFdAu30h1lgOeLy:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/7b34190fe40f6d25cc274653dd21219d9f354de1/v19.1/gssapi_authentication.md#L84)):*
<details><summary><i>Previously, rolandcrosby (Roland Crosby) wrote…</i></summary><blockquote>

That seems fine. You're right that the tables/DBs have to exist; maybe you could just show like `GRANT ALL ON DATABASE defaultdb TO carl`?
</blockquote></details>

Done.

File: v19.1/gssapi_authentication.md [hunk 1]
```diff
@@ -86,6 +81,20 @@ To use Kerberos authentication with CockroachDB, configure a Kerberos service pr
...
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    > CREATE USER carl;
+    ~~~
+
+    Grant privileges to the user:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    > GRANT ALL ON DATABASE defaultdb TO carl;;
```
Comment (mjibson) [inline v19.1/gssapi_authentication.md]: double `;`

File: v19.1/gssapi_authentication.md [hunk 2]
```diff
@@ -86,6 +81,20 @@ To use Kerberos authentication with CockroachDB, configure a Kerberos service pr
 
       The `include_realm=0` option is required to tell CockroachDB to remove the `@DOMAIN.COM` realm information from the username. We don't support any advanced mapping of GSSAPI usernames to CockroachDB usernames right now. If you want to limit which realms' users can connect, you can also add one or more `krb_realm` parameters to the end of the line as a whitelist, as follows: `host all all all gss include_realm=0 krb_realm=domain.com krb_realm=corp.domain.com`
 
+8. Create CockroachDB users for every Kerberos user. For example, if one of your Kerberos users is Carl, create a CockroachDB user named `carl`:
```
Comment (mjibson) [inline v19.1/gssapi_authentication.md]: Maybe make the username `carl@realm.com` to illustrate that they need to remove the `@realm.com` part.

**Comment by Amruta-Ranade** [comment]
@jseldess Ready for your review!

**Pull request merged**